doc/source/tutorials/data/ntwk1.s2p

```diff
@@ -1,4 +1,4 @@
-! Created Thu Nov 11 11:09:06 2010
+! Created Thu Nov 11 11:09:06 2010
```

Comment:
same comment, would be nice to remove unchanged files from the PR

Reply:
Reverted all files that were only different in how they handled line feeds (i.e. CRLF vs LF)